tests/firecrackerdrivertest.c

```diff
@@ -0,0 +1,302 @@
+/*
+ * Copyright (C) 2021, Amazon, Inc.
+ *
+ * This library is free software; you can redistribute it and/or
+ * modify it under the terms of the GNU Lesser General Public
+ * License as published by the Free Software Foundation; either
+ * version 2.1 of the License, or (at your option) any later version.
+ *
+ * This library is distributed in the hope that it will be useful,
+ * but WITHOUT ANY WARRANTY; without even the implied warranty of
+ * MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the GNU
+ * Lesser General Public License for more details.
+ *
+ * You should have received a copy of the GNU Lesser General Public
+ * License along with this library;  If not, see
+ * <http://www.gnu.org/licenses/>.
+ */
+
+#include <config.h>
+
+#include "testutils.h"
+
+#include "virlog.h"
+
+#ifdef WITH_FC
+
+#include "vircommand.h"
+
+# define VIR_FROM_THIS VIR_FROM_NONE
+
+VIR_LOG_INIT("tests.firecrackerdrivertest");
+
+static int
+testDriverConnection(const void *opaque G_GNUC_UNUSED)
```

Comment:
why do we run this, since in `testLifecycle` we are already testing this?
I think we can remove it

Reply:
I'm fine keeping it then